tyk-headless/values.yaml

```diff
@@ -24,34 +24,47 @@ secrets:
   useSecretName: ""
 
 redis:
-  # The addrs value will allow you to set your Redis addresses. If you are
-  # using a redis cluster, you can list the endpoints of the redis instances
-  # or use the cluster configuration endpoint.
+  # The addrs value will allow you to set your Redis addresses. 
+  #
+  # If you are using Redis (e.g. Bitnami Redis at bitnami/redis) then enter single
+  # endpoint.  If using "Redis Sentinel" update the port number (typically 26379).
+  #
+  # If using a "Redis Cluster" (e.g. bitnami/redis-cluster), you can list 
+  # the endpoints of the redis instances or use the cluster configuration endpoint.
+  # 
   # Default value: redis.{{ .Release.Namespace }}.svc.cluster.local:6379
+  #
   # addrs:
-  #   - redis.tyk.svc.cluster.local:6379
-  #   This is the the DNS name of the redis as set when using Bitnami
+  #   Example using bitnami/redis
+  #   - tyk-redis.tyk.svc.cluster.local:6379
+  #   Example using bitnami/redis with Sentinel
+  #   - tyk-redis.tyk.svc.cluster.local:26379
+  #   Example using bitnami/redis-cluster
   #   - tyk-redis-master.tyk.svc.cluster.local:6379
 
   # Redis password
-  # If you're using Bitnami Redis chart please input your password in the field below
+  # If you're using Bitnami Redis chart (e.g. bitnami/redis) please input 
+  # your password in the field below
   # pass: ""
 
   # Enables SSL for Redis connection. Redis instance will have to support that.
   # Default value: false
   # useSSL: true
 
-  # The enableCluster value will allow you to indicate to Tyk whether you are
-  # running a Redis cluster or not.
-  # enableCluster: true
+  # If using "Redis Cluster" set enableCluster to true
+  # (e.g. if using bitnami/redis-cluster)
+  # enableCluster: false
```

Comment:
Please leave this part as it is for now.

Reply:
Just to be clear, leave `enableCluster: false` for now?

Reply:
I will return it back to true as per the original.

Reply:
Please see revised commits.